shell/tools/detect-unused-files.ts

```diff
@@ -65,6 +65,8 @@ const detect = () => {
   (result.unused || []).forEach((item) => {
     console.log(item);
   });
+  console.log(`unused export count: ${result.unusedExport.size}`);
+  console.log(result.unusedExport);
```

Comment:
This script only do log but not modify original file directly, right ?

Reply:
yep, just remind people which ones are unused, delete operation should be done manually